alacritty/src/config/bindings.rs

```diff
@@ -297,6 +301,7 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
     let mut bindings = bindings!(
         KeyBinding;
         Copy;  Action::Copy;
+        Copy,  +TermMode::VI; Action::Copy;
```

Comment:
Why are there two bindings for Copy here? A binding for all modes is always also active for vimode?

Did you mean to use `ClearSelection` as action here?